### PR TITLE
Static pie chart labels

### PR DESCRIPTION
**Pull request by alxnddr**
## Changes

Adds data labels to the static pie chart.

## Screenshots

Screen Shot 2022-10-17 at 11 39 51 PM

## How to verify

- Run Maildev/Mailhog and setup email settings on your instance
- Create a pie chart
- Enable "Show data labels" setting
- Add it on a dashboard and send an email subscription
- Ensure static pie chart includes data labels

### CODE REVIEW

**Review by ranquild — APPROVED** [review]
I had to merge `master` because otherwise dashboards didn't work 
Screenshot 2022-10-20 at 14 45 17 Otherwise works as expected

File: test/metabase/pulse/render/body_test.clj [hunk 1]
```diff
@@ -603,7 +603,7 @@
                                     x))
                                 html-tree))]
     (testing "Renders without error"
-      (let [rendered-info (render [["Doohickey" 75] ["Widget" 25]])]
```
Comment (tsmacdonald) [inline test/metabase/pulse/render/body_test.clj:606]: `show_values` or `show_labels` ?

File: test/metabase/pulse/render/js_svg_test.clj [hunk 2]
```diff
@@ -298,11 +298,12 @@
 (deftest categorical-donut-test
   (let [rows [["apples" 2]
               ["bananas" 3]]
-        colors {"apples" "red" "bananas" "yellow"}]
+        colors {"apples" "red" "bananas" "yellow"}
+        settings {:show_values true}]
```
Comment (tsmacdonald) [inline test/metabase/pulse/render/js_svg_test.clj:302]:
```suggestion
        settings {:show_labels true}]
```

**Comment by tsmacdonald** [comment]
@adam-james-v thoughts on using `metabase.shared.moedls.visualization-settings` magic so that we can use `show-labels` on the Clojure side and `show_labels` on the FE side and—I think—get a bit more safety around using the right key to begin with? I'm a little nervous about the existing `show_labels` / `show_values` mixup

**Review by tsmacdonald — APPROVED** [review]

**Pull request merged**